app/vmui/packages/vmui/src/hooks/useQuickAutocomplete.ts

```diff
@@ -0,0 +1,24 @@
+import useEventListener from "./useEventListener";
+import { useQueryDispatch } from "../state/query/QueryStateContext";
+import { useCallback } from "react";
```

Comment:
Please switch to `preact/compat`:
```suggestion
import { useCallback } from "preact/compat";
```

Keeps consistency and avoids mixing React with Preact.